src/datanode/src/config.rs

```diff
@@ -66,8 +78,8 @@ pub struct StorageConfig {
     pub global_ttl: Option<Duration>,
     /// The working directory of database
     pub data_home: String,
-    #[serde(flatten)]
-    pub store: ObjectStoreConfig,
+    pub default_store: ObjectStoreConfig,
+    pub custom_stores: Vec<ObjectStoreConfig>,
```

Comment:
What about `providers`? Then the config file loos like:

```toml
[storage.providers]
type = 'S3'
....
[storage.providers]
type = 'Gcs'
....
```

Reply:
Consider the following names for this config
- providers
- services
- backends

Reply:
@evenyag 
I selected the `providers` based on @killme2008  suggestion. Which name do you prefer?